obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+  mp_starcat_time: 2013:312:06:21:15.039
+  status: 1
+- agasc_id:
+  - 898026352
+  comments: Venus observation
+  mp_starcat_time: 2013:312:06:21:15.039
+  status: 1
+- agasc_id:
+  - 897995504
+  comments: Venus observation
+  mp_starcat_time: 2013:312:08:58:05.039
+  status: 1
+- agasc_id:
+  - 897996696
+  comments: Venus observation
+  mp_starcat_time: 2013:312:08:58:05.039
+  status: 1
+- agasc_id:
+  - 898002928
+  comments: Venus observation
+  mp_starcat_time: 2013:312:08:58:05.039
+  status: 1
+- agasc_id:
+  - 898026352
+  comments: Venus observation
+  mp_starcat_time: 2013:312:08:58:05.039
+  status: 1
+- agasc_id:
+  - 897996696
+  comments: Venus observation
+  mp_starcat_time: 2013:312:11:34:55.039
+  status: 1
+- agasc_id:
+  - 897999536
+  comments: Venus observation
+  mp_starcat_time: 2013:312:11:34:55.039
+  status: 1
+- agasc_id:
+  - 898002928
+  comments: Venus observation
+  mp_starcat_time: 2013:312:11:34:55.039
+  status: 1
+- agasc_id:
+  - 898026352
+  comments: Venus observation
+  mp_starcat_time: 2013:312:11:34:55.039
+  status: 1
+- agasc_id:
+  - 897995504
+  comments: Venus observation
+  mp_starcat_time: 2013:312:14:11:45.039
+  status: 1
+- agasc_id:
+  - 897999536
+  comments: Venus observation
+  mp_starcat_time: 2013:312:14:11:45.039
+  status: 1
+- agasc_id:
+  - 898004384
+  comments: Venus observation
+  mp_starcat_time: 2013:312:14:11:45.039
+  status: 1
+- agasc_id:
+  - 898026352
+  comments: Venus observation
+  mp_starcat_time: 2013:312:14:11:45.039
+  status: 1
+- agasc_id:
+  - 897988944
+  comments: Venus observation
+  mp_starcat_time: 2013:312:16:48:35.039
+  status: 1
+- agasc_id:
+  - 897995504
+  comments: Venus observation
+  mp_starcat_time: 2013:312:16:48:35.039
+  status: 1
+- agasc_id:
+  - 898004384
+  comments: Venus observation
+  mp_starcat_time: 2013:312:16:48:35.039
+  status: 1
+- agasc_id:
+  - 898026352
+  comments: Venus observation
+  mp_starcat_time: 2013:312:16:48:35.039
+  status: 1
+- agasc_id:
+  - 897988944
+  comments: Venus observation
+  mp_starcat_time: 2013:312:19:25:25.039
+  status: 1
+- agasc_id:
+  - 897995504
+  comments: Venus observation
+  mp_starcat_time: 2013:312:19:25:25.039
+  status: 1
+- agasc_id:
+  - 898004384
+  comments: Venus observation
+  mp_starcat_time: 2013:312:19:25:25.039
+  status: 1
+- agasc_id:
+  - 898004384
+  comments: Venus observation
+  mp_starcat_time: 2013:312:22:02:15.039
+  status: 1
+- agasc_id:
+  - 898124000
+  comments: Venus observation
+  mp_starcat_time: 2013:312:22:02:15.039
+  status: 1
+- agasc_id:
+  - 898125104
+  comments: Venus observation
+  mp_starcat_time: 2013:312:22:02:15.039
+  status: 1
+- agasc_id:
+  - 898004384
+  comments: Venus observation
+  mp_starcat_time: 2013:313:00:39:05.039
+  status: 1
+- agasc_id:
+  - 898135232
+  comments: Venus observation
+  mp_starcat_time: 2013:313:00:39:05.039
+  status: 1
+- agasc_id:
+  - 898125104
+  comments: Never acquired. Set mag to MAXMAG
```

Comment:
Obsid 53250 was really a Venus observation too, so star 898125104 should be "forgiven".

Reply:
I was taking the list of Venus observations from [mica.bad_obsids](https://github.com/sot/mica/blob/master/mica/bad_obsids.py), and obsid 53250 is not there.

I just went to the starcheck page, and it is not marked as a Venus observation. There are other obsids marked as Venus observations in the same page. Am I missing something?

None of the stars you say should be "forgiven" are in Venus observations listed in mica.obsids.

I intend to make a commit removing all Venus star-observations and adding the observations themselves (which will remove other stars not included here). Do you want me to wait until you are done reviewing?

Reply:
Right.  I had just looked up 53250, seen that it is a "Special Case ER" right after a Venus observation and at the same attitude.  It would be a different exercise to see if Venus is still on the CCD.  I assume when I made the list of mica Venus observations I did not look for ERs as I figured we would not dwell on Venus during an ER.

Regarding "There are other obsids marked as Venus observations in the same page. Am I missing something?"  I'm not sure what you mean?

Reply:
"None of the stars you say should be "forgiven" are in Venus observations listed in mica.obsids."  And did I say this about more than one star?  I'll look again.